dags/main_summary.py

```diff
@@ -210,6 +210,22 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/client_count_daily_view.sh",
     dag=dag)
 
+main_summary_glue = EMRSparkOperator(
```

Comment:
This probably doesn't need to be an EMR instance, saving us a bit of bootstrap time. No change needed, just a consideration for the future.